src/common/options.cc

```diff
@@ -7059,6 +7059,10 @@ std::vector<Option> get_rgw_options() {
     .set_default(true)
     .set_description("Should RGW verify the vault server SSL certificate."),
 
+    Option("rgw_crypt_vault_ssl_cert", Option::TYPE_STR, Option::LEVEL_ADVANCED)
+    .set_default("")
+    .set_description("Path for customed ca certificate for accessing vault server"),
```

Comment:
"custom" not "customed"

Reply:
Also, maybe just say "Path to CA certificate for accessing Vault server"--if we provide a CA cert, it's just a requirement of the setup, maybe not anything actually special or custom.